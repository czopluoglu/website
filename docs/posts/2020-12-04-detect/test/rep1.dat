11000101011111101111
11101100110001111110
10110101001011100011
01110101011010100010
10000000011100001010
10000011100110101110
01001010111000011111
00000001101010101111
00000001001010100101
10001101011000100100
10110001011110110010
10000000101110010110
10110001101011000001
00001000001001101011
11111001111110101101
00010100111110111110
10000101101100001101
11010101100010001110
00000001011001100001
10111101110100101111
11011010101011111110
10000001001000111100
11101111011011010110
00000010101010101111
11000000010010100001
10000000001011100011
10111010111100000010
01100101101111011111
11000101111010011100
11101111111010100110
11100011111101100110
10000011100110101100
10100001100011010110
00010011001011000110
01110110011001010100
01000010100010101101
00110100100111011111
01001000101110100111
00000000000100000100
00011000011111010010
01110001111000011111
00000000010100010111
10101111001111011111
10001110111111100111
11101100011011110111
10001101111111110001
11000101011110001011
00001100101010101001
00000011011111110111
10110000101110111111
10011111111110000011
01010101101101100110
11101110011011111111
11111111110110011010
01001111111111011011
00100010101001111011
11111001111011000111
00100111111011011011
11001000000110110111
01001101001010110010
10101001101000110001
11100101110011101011
01010110110100000111
00000000011011000011
00101010010100111111
10001111110000111001
00110001110000000011
01111111111010111110
11001111001111111100
11001111101010111101
00111011101001101001
00010111110110000011
00001001011111011111
00000000001110111011
00011101011111100100
11101001011010101110
10011101001010101111
01010001111000011100
00000000101010100111
10001111011111101111
10010000011111111111
10001100011111110011
00110001001110110111
10101101101010111111
11101001101011001111
11100000101110000001
01100101010000000101
10001000110001000010
01101010110000111011
01001011011110101111
00001100011111011110
11111110011010110110
01001001100101011111
01111000110010000011
01011000111110101111
01111101111110111101
10110101101111101111
10000101111100101010
00011101110001010011
01111000011100001011
01110101001110110111
11110111111110100011
10111101101011011111
00000001011000101010
01101001111111101111
00001001100000100001
11001100101010000110
10000001110110100110
01101011000011011000
00110100001011011000
11000010111100101110
10011001010111110010
11111001001001010111
01000011011111111101
10011111011111010011
10110111110101111111
10111101111101100010
10111011101111111111
01101000111111001111
11111010101011101101
10111101001101111111
11100101011011101110
10000000001000011110
10011101011101100010
10000001111111111110
00101000001000101100
10001100001011111001
11111010111101001011
00011001101000000011
00101011011000110111
00000101101001111010
00101001011001100111
10000000111101101010
10000001111111100110
10011000111011111111
01001011111110101111
10011011110011110110
00101100001010110011
01111101100000101010
00000001000110000010
11011100110101000111
10001001010110101001
11101111111001000001
11011101111110111111
00010001011001110101
11100111111111111111
01100000001011010001
01011101110000100010
01000001000010101010
10001011001100101101
11111101001000011100
11001111101110111111
10111101001101111110
10000110101001001011
10001100010000010111
00000001001010010011
01110101011111101110
01010001111110111111
01110100001000001111
10000100111011011010
10100101101010101011
00100001000011010010
01000000111101001110
00101000011000001010
11000101011011111000
01101101100000111000
11110000111010001011
01011101010011110010
10111000110001100101
11111001011111111111
01110011110011110101
11000101011110111111
11011100010011100110
10110111111100111111
00100101111111101111
10110101111010010001
00110010111000011011
10000000011010011001
10111001111101111111
10101111011111111110
11111111110000000011
11110000101001110000
01111001111111101010
01010001111111100100
00000100001001000010
10110111111111111011
10011101101011101111
11100111111110000101
11010000010000000010
01000101011110100010
10000111001111111010
01101001001001001100
11111110110011110110
10111011001010110100
11111111111110000110
11001111011111111110
00001001001011000011
11010000101101111011
00000000100011001110
00110001010100100101
10111001101000001010
01110001001011100110
00111000101111111011
11000000011001100011
11001001001001000001
11000001110100001001
00010111010111010101
01101011111000111000
10000111001100110010
00110111111110111110
00011000001000101101
10100001000010100011
10001100011000000000
10110011100111100111
10011111111111111110
11011111111111111111
11110111110010010010
01000000011101111110
01010011000000011010
10101000011011000011
11100001001000001110
10001101100000100011
01110000001100101010
01100111111011111111
10101000111100110011
00010101110110101101
01001000010010000010
00011001111111101111
11111101101100101111
11011010011110110111
11011111110111001110
00101101011000100111
01010111110001100110
11011111111111111111
01010010101111001111
01101001100111101111
00110000001010110011
01010000011110001101
11101111101111111111
11101101010001000111
10101000100110111111
01110101111101111111
10000001001010111111
10110011111101000010
00110001010010000110
01101001111011101111
11101010101111110000
01001111001100101101
10110001110100101010
10000101101101111011
