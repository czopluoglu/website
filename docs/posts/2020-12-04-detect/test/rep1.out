-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep1.dat                                           

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.80

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.50
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  4
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.7276
                   IDN index value:   0.6842
                           Ratio r:   0.5866

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    2    2    2    1    1    2    1    2    2
    3    3    3    4    3    3    3    3    4    3

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    5    6    8

   -----------CLUSTER  2-------------

    2    3    4    7    9   10

   -----------CLUSTER  3-------------

   11   12   13   15   16   17   18   20

   -----------CLUSTER  4-------------

   14   19
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d++----++--+-+----+-
   +d+-++-+-+---+---++-
   ++d++--+-+--+-+---++
   --+d-+++---+-+--+---
   -++-d+-+++-----++---
   -+-++d+-+-----+--+-+
   ---+-+d-++-+-------+
   +++++--d++-++-+-+---
   +---++++d--+---+-+-+
   -++-+-++-d-+------++
   ----------d+++++-++-
   +--+--+++++d++-+++-+
   --+----+--++d++-++++
   ++-+------+++d++++--
   --+--+-+--+-++d+--+-
   ----+---+-++-++d++--
   ---++--+---+++-+d+--
   -+---+--+-++++-++d--
   +++------++-+-+---d+
   --+--++-++-++-----+d
 
 No cross validation for this DETECT run
